### PR TITLE
feat: implemented show details button logic; kinda deep-linking

### PR DESCRIPTION
**Pull request by Toxer10**
## Changes

-  implemented show details button logic; kinda deep-linking

### CODE REVIEW

File: src/components/molecules/ExecutionStepsList/ExecutionStepsList.tsx [hunk 1]
```diff
@@ -47,18 +52,40 @@ const ExecutionStepsList: React.FC<ExecutionStepsListProps> = props => {
...
   };
 
-  const onShowTestClick = (testName: string) => {
-    return navigate(`/dashboard/tests?textSearch=${testName}`);
+  const onShowClick = (step: any) => {
+    const {executionName} = step;
+
+    if (iconSet === 'default') {
+      const {
+        execution: {id},
```
Comment (pavloburchak) [inline src/components/molecules/ExecutionStepsList/ExecutionStepsList.tsx:60]: Does `execution` always exist here?

File: src/components/organisms/DashboardContent/DashboardContent.tsx [hunk 2]
```diff
@@ -138,29 +141,43 @@ const DashboardContent: React.FC<any> = props => {
       return;
     }
 
-    if (contentProps.data === null || contentProps.data?.results === []) {
-      dispatch(setData([]));
-
-      if (canSelectRow) {
-        dispatch(setSelectedRecord({selectedRecord: null}));
+    if (contentProps.data || contentProps.data?.length) {
```
Comment (pavloburchak) [inline src/components/organisms/DashboardContent/DashboardContent.tsx:144]: You can maybe create a condition variable and pass it to only one `if`. There is unnecessary to put 3 `ifs` in a row without `else`

File: src/components/molecules/ExecutionStepsList/ExecutionStepsList.tsx (hunk 1 above)
Comment (Toxer10) [reply src/components/molecules/ExecutionStepsList/ExecutionStepsList.tsx:60]: Yes

**Review by pavloburchak — APPROVED** [review]

File: src/components/organisms/DashboardContent/DashboardContent.tsx (hunk 2 above)
Comment (Toxer10) [reply src/components/organisms/DashboardContent/DashboardContent.tsx:144]: Resolved!

**Pull request merged**